packages/jsx/src/index.ts

```diff
@@ -84,11 +84,13 @@ const walkLinkedList = (ctx: Ctx, el: JSX.Element, list: Atom<LinkedList<LLNode<
 }
 
 export const reatomJsx = (ctx: Ctx, DOM: DomApis = globalThis.window) => {
-  const StylesheetId = 'reatom-jsx-styles'
   let styles: Rec<string> = {}
-  let stylesheet: HTMLStyleElement | undefined
+  let stylesheet: HTMLStyleElement = DOM.document.createElement('style')
+  stylesheet.id = 'reatom-jsx-styles'
   let name = ''
 
+  DOM.document.head.appendChild(stylesheet)
```

Comment:
If a user will not using css prop there will unnecessary style element in their DOM.

Reply:
I made this change because dynamically adding a style tag can introduce style order. When changing the order, random bugs with styles can occur.

Reply:
Agree. Please, add an option to the fabric to get an ability to disable the element insertion.

Reply:
Replaced Record with Map, because Map is more productive when dynamically adding and reading